packages/atlas/src/components/_video/VideoTilePublisher/VideoTilePublisher.tsx

```diff
@@ -47,11 +47,19 @@ export const DELAYED_FADE_CLASSNAME = 'delayed-fade'
 
 export const VideoTilePublisher: FC<VideoTilePublisherProps> = memo(
   ({ id, onEditClick, onDeleteVideoClick, onReuploadVideoClick, onMintNftClick, titlesInSync }) => {
+    const [videoTitleMap, setVideoTitleMap] = useState('')
     const { video, loading } = useFullVideo(id ?? '', {
       skip: !id,
       onError: (error) => SentryLogger.error('Failed to fetch video', 'VideoTilePublisher', error, { video: { id } }),
-      pollInterval: titlesInSync?.length ? 1000 : undefined,
+      pollInterval: titlesInSync?.length && titlesInSync.includes(videoTitleMap) ? 1000 : undefined,
```

Comment:
Could we increase the interval a bit here, to three seconds at least? I don't think we should refetch that frequently. 

Also, let's store this in some global variable to not repeat this in MyVideosView.